tsconfig.json

```diff
@@ -3,14 +3,14 @@
     "esModuleInterop": true,
     "exactOptionalPropertyTypes": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["ES2020"],
-    "module": "CommonJS",
-    "moduleResolution": "node",
+    "lib": ["ES2023"],
```

Comment:
We can use a ECMAScript version here that's newer than the target, in which case TypeScript will transpile newer features if necessary.